Source/WebCore/rendering/RenderGrid.cpp

```diff
@@ -1808,20 +1808,22 @@ WeakPtr<RenderBox> RenderGrid::getBaselineChild(ItemPosition alignment) const
     ASSERT(alignment == ItemPosition::Baseline || alignment == ItemPosition::LastBaseline);
     const RenderBox* baselineChild = nullptr;
     unsigned numColumns = currentGrid().numTracks(ForColumns);
-    unsigned numRows = currentGrid().numTracks(ForRows);
-
+    auto rowIndexDeterminingBaseline = alignment == ItemPosition::Baseline ? 0 : currentGrid().numTracks(ForRows) - 1;
```

Comment:
This is the first or last row of the grid, not necessarily the first or last row that contains at least one item. Is that a bug?

Reply:
Yeah I noticed this as well and think it might be, but I decided just to match what we are doing for first baseline for now. I was planning on testing this and following up on this in an additional patch